app/main/views.py

```diff
@@ -265,9 +265,12 @@ def history():
     current_app.logger.info('Finished querying')
 
     query_has_results = True if events_query.first() else False
-
+    all_info = calculate_hours_worked(session['email'], session['first_date'], session['last_date'])
+    total_hours = all_info['total_hours'] if all_info else "Each clock in must have corresponding clock out"
```

Comment:
We should probably tell the user what steps to take if we couldn't calculate the total number of hours.